docs/source/variables.adoc

```diff
@@ -1,2 +1,2 @@
-:minishift-version: unset
-:openshift-version: unset
+:minishift-version: 1.0.0-rc.1
```

Comment:
Sure, as I said, it most likely got added accidentally prior it being part of .gitignore. I did not notice until now. You basically did a build, the files got updated and you committed them. 

What I am saying is, that the file should be removed from the repo now.